project/firstproject/src/main/resources/templates/loggedIn.html

```diff
@@ -64,5 +70,35 @@ <h4 class="modal-title">Add Question Form</h4>
         </div>
     </div>
 </div>
+
+
+<table class="table">
+    <thead class="thead-dark">
+    <tr>
+        <th scope="col">#</th>
+        <th scope="col">Title</th>
+        <th scope="col">Question</th>
+        <th scope="col">Creator</th>
+        <th scope="col">Date Created</th>
+        <th scope="col"></th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr th:each="q: ${listQuestions}">
+        <td th:text="${q.id}"></td>
+        <td th:text="${q.title}"></td>
+        <td th:text="${q.question}"></td>
+        <td th:text="${q.creator}"></td>
+        <td th:text="${q.createdDate}"></td>
+        <td>
+            <a th:href="@{/answer/view-answer}" class="btn btn-info"><i
```

Comment:
Send the ID of the question here so you will be able to see answers for a particular question only.